content/automate/ManageIQ/System/Notification/Email.class/__methods__/miq_provision_complete.rb

```diff
@@ -0,0 +1,48 @@
+#
```

Comment:
@billfitzgerald0120 
Since this is a new method can we convert it to the new format.

Reply:
Yes, converting to new format now